pxr/usdImaging/usdImagingGL/engine.cpp

```diff
@@ -1153,7 +1153,13 @@ UsdImagingGLEngine::SetRendererSetting(TfToken const& id, VtValue const& value)
     }
 
     TF_VERIFY(_renderDelegate);
-    _renderDelegate->SetRenderSetting(id, value);
+    // usdviewq passes a double as VtValue since Python has no 'float' type
+    if (value.IsHolding<double>()) {
+        _renderDelegate->SetRenderSetting(
+            id, VtValue(static_cast<float>(value.UncheckedGet<double>())));
```

Comment:
There is builtin support for value casting:` VtValue::Cast<float>(value)`

Also, if we are going to do this here, instead of in the python bindings, then presumably the goal is to fully protecct render delegates such that they only receive the declared three value types.  If so, then should we not also add conversion-guards for `half`, `long`, `TfToken`, and issue a coding error if we see any other type?